Movie.java

```diff
@@ -18,16 +18,17 @@ public class Movie implements Serializable{
 	private ArrayList<Cinema> cinemas;
 	private int runtime;  //Duration of the movie in (MINS)
 	private LocalDate movieReleaseDate;
+	private double totalSales;
 
 	/*
 	 * No abstract methods atm
 	 */
-	
-	
+
+
 	//Constructor
 	public Movie(int id, String title, MovieType type, MovieStatus status, String synopsis, String director, ArrayList<String> casts,
-			String rating, int runtime, LocalDate movieReleaseDate) {
-		
+			String rating, int runtime, LocalDate movieReleaseDate, double totalSales) {
```

Comment:
For all the Movie class updates, please check in with @ChooJinCheng since he was the original author and has the best idea about his implementations. Ensure you're both on the same page about the new updates.

Reply:
I've have updated @ChooJinCheng with the changes and have updated accordingly.